.gitignore

```diff
@@ -1,2 +1,36 @@
-node_modules/
-.env
\ No newline at end of file
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.package-lock.json
+
+# testing
+/coverage
+.vercel
+
+# production
+/build
+
+# misc
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# local env files
+.env*.local
+.env
+
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
+next-env.d.ts
+gh-pages/.DS_Store
```

Comment:
this is redudant with the line above .DS_Store and should be part of the gobal git ignore imho